Last-Update: 2016-08-29
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: change audio codec (priority) order to match priority of
 gstreamer plugins packages alternatives:
  base(vorbis); good(wavpack, wav, flac, speex); ugly(mp3); bad(opus).

--- a/xpra/sound/gstreamer_util.py
+++ b/xpra/sound/gstreamer_util.py
@@ -201,12 +201,10 @@
         SPEEX       : 0,
        }
 
 CODEC_ORDER = [
-    OPUS, OPUS_OGG, VORBIS_MKA, VORBIS_OGG, VORBIS,
-    MP3, FLAC_OGG, AAC_MPEG4,
-    WAV_LZ4, WAV_LZO, WAV, WAVPACK,
-    SPEEX_OGG, VORBIS, OPUS_MKA, FLAC, MP3_MPEG4,
+    VORBIS_MKA, VORBIS, FLAC, FLAC_OGG, WAV_LZ4, WAV_LZO, WAV, WAVPACK,
+    SPEEX_OGG, MP3, AAC_MPEG4, MP3_MPEG4, OPUS_OGG, OPUS, OPUS_MKA
     ]
 
 
 gst = None
